dashboard/app/controllers/aichat_controller.rb

```diff
@@ -251,4 +257,10 @@ def check_message_safety
       project_id
     end
   end
+
+  private def should_throttle?
+    id = current_user&.id || session.id
```

Comment:
Users will always be logged in as-is, right? The `session.id` is just in case we were to open to non-logged in usage?

Reply:
Yep that's right. I copied this from another throttling example, but we should expect to always have a user ID, at least at launch.